V8Helpers.h

```diff
@@ -440,3 +440,8 @@ namespace V8
 		V8_CHECK(!baseObject.IsEmpty(), "Failed to bind base object"); \
 		resource->BindEntity(info.This(), baseObject.Get()); \
 	}
+
+#define V8_MARK_ENTITY_FROM_JS(entity) \
```

Comment:
I'd name it as `V8_IMPRINT_ENTITY`, which means exactly what this definition can be used for